src/Views/Appearance.vala

```diff
@@ -43,12 +43,18 @@ public class PantheonShell.Appearance : Gtk.Grid {
 
     private Granite.Widgets.ModeButton text_size_modebutton;
 
-    construct {
-        column_spacing = 12;
-        halign = Gtk.Align.CENTER;
-        row_spacing = 6;
-        margin_start = margin_end = 6;
+    public Appearance () {
+        Object (
+            column_spacing: 12,
+            halign: Gtk.Align.CENTER,
+            margin_bottom: 24,
+            margin_end: 6,
+            margin_start: 6,
+            row_spacing: 6
+        );
```

Comment:
Hrm, I'm not like a massive fan of putting properties in the object call that weren't passed into the constructor. But also, shouldn't we put all the same margins on the stack itself instead of in each view?

Reply:
@danrabbit yeah you might be right re: moving the properties. Re: the margins, that was my initial thought as well, but it looks like we intentionally avoided it on the Wallpapers view because of the sort of bottom toolbar design.